updater.ps1

```diff
@@ -0,0 +1,20 @@
+$channel = "https://pkgs.tailscale.com/stable/"
+
+$url = $channel + ((Invoke-WebRequest $channel).links | where-object { $_.href -like "*.exe" }).href

```

Comment:
This seems fragile. How about you use https://pkgs.tailscale.com/stable/?mode=json&os=windows instead?

Reply:
I knew there was a JSON mode, but I couldn't find any documentation on it.

Reply:
Updated.